sentry-apollo/build.gradle.kts

```diff
@@ -8,6 +8,7 @@ plugins {
     id(Config.QualityPlugins.errorProne)
     id(Config.QualityPlugins.gradleVersions)
     id(Config.BuildPlugins.buildConfig) version Config.BuildPlugins.buildConfigVersion
+    id(Config.QualityPlugins.androidLint)
```

Comment:
Apollo is also available for non Android, not sure how we would add it

Reply:
@adinauer what do you mean? This is the Apollo module and I'm adding the `lint` to it, so it's alright.

Reply:
Yes I was trying to say that we probably can't just add the android plugin to it to gain `android {}` in there as this can also be used by non Android applications.

Reply:
I managed to config using `configure<Lint>` but it still does not detect old APIs compatibility.

Reply:
Could we simply run grep on the logs instead and check for minApi warnings?

Reply:
But which logs do you want to run grep on? You need to be able to run lint first :)

Reply:
I thought it works but we can't customize it to fail the build for minSdk violations.

Reply:
the Linter runs already, but it does not check for `newApi`.

Reply:
https://github.com/open-toast/gummy-bears
This can help
have you used/tried this @romtsn @markushi ?